swagger-parser/2.0.20.wso2v2/pom.xml

```diff
@@ -0,0 +1,193 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+ ~ Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
should be 2023

Reply:
fixed with https://github.com/wso2/orbit/pull/938/commits/bbc74c4bd027d2c656b1752e44de28ad0e70548f